src/platform/utilities/feature-toggles/helpers.js

```diff
@@ -0,0 +1,28 @@
+const Cookies = require('js-cookie');
```

Comment:
it failed when I used import

Reply:
It's worth trying to change to `import`, but I expect it won't work since this is imported in the build script, which is entirely Node and not run through Babel.

Reply:
ooh